lib/rubocop/cop/registry.rb

```diff
@@ -125,10 +125,14 @@ def enabled(config, only, only_safe = false)
 
       def enabled?(cop, config, only_safe)
         cfg = config.for_cop(cop)
+
+        # cfg['Enabled'] might be a string `none`, which is considered disabled
```

Comment:
I think `pending` or `new` might be a better name for this. Nil might be an option as well, although it's not very descriptive.